src/probnum/filtsmooth/optim/_iterated_component.py

```diff
@@ -61,7 +61,9 @@ def __init__(
     ):
         self._component = component
         self.stopcrit = (
-            _stoppingcriterion.StoppingCriterion() if stopcrit is None else stopcrit
+            _stopping_criterion.FiltSmoothStoppingCriterion()
+            if stopcrit is None
+            else stopcrit
```

Comment:
multi-lines of these if-else constructions are not recommended due to readability.